docker-compose.yml

```diff
@@ -47,6 +47,45 @@ services:
     networks:
       - cw-network
 
+  commonwealth:
```

Comment:
Just a quick note on this, `docker-compose up` is unchanged, it will not run the app via docker. This will only be run if you pass in the `--profiles with-app` build argument